guides/release/routing/rendering-a-template.md

```diff
@@ -25,9 +25,9 @@ the template you want to render instead.
 ```javascript {data-filename=app/routes/posts.js}
 import Route from '@ember/routing/route';
 
-export default Route.extend({
-  templateName: 'posts/favorite-posts'
-});
+export default class PostsRoute extends Route {
+  templateName = 'posts/favorite-posts';
```

Comment:
Another place where this ES6 syntax moves a property from the prototype to each instance. Given that this has bitten us before (i.e., computed properties defined like `foo = computed(...)`) we should get confirmation that this is, and will continue to be supported.

Reply:
What are you worried won't be supported?

Reply:
Let's iterate on this after discussion so this can be merged promptly.